src/functions.js

```diff
@@ -205,7 +205,7 @@ defineFunction("\\rule", {
     };
 });
 
-defineFunction("\\kern", {
+defineFunction(["\\kern", "\\mkern"], {
```

Comment:
According to TeXbook on pg. 168, `\mkern` only accepts measurements given in `mu` units and `\kern` cannot accept `mu` units.  Can you add a comment describing this deviation from spec?  We'll probably want to enforce these constraints at some point, but it doesn't seem critical at the moment.